systems/plants/RigidBodyFlatTerrain.m

```diff
@@ -3,13 +3,18 @@
 %  everywhere
   
   methods 
-    function obj = RigidBodyFlatTerrain()
+    function obj = RigidBodyFlatTerrain(varargin)
+      % Construct a flat terrain map at a given z height (default 0)
+      p = inputParser();
```

Comment:
There's still some argument to be made for not changing the standard we're currently using, but it would be nice to simplify our syntax to:

```
v = r.constructVisualizer('slider', true)
```

without losing the ability to do:

```
v = r.constructVisualizer(struct('slider', true));
```